client/client.go

```diff
@@ -259,6 +259,98 @@ type (
 	// WARNING: Worker versioning is currently experimental
 	TaskQueueReachability = internal.TaskQueueReachability
 
+	// UpdateWorkerVersioningRulesOptions is the input to Client.UpdateWorkerVersioningRules.
+	// WARNING: Worker versioning-2 is currently experimental
+	UpdateWorkerVersioningRulesOptions = internal.UpdateWorkerVersioningRulesOptions
```

Comment:
This is gonna sound annoying, but you'll want to copy this Godoc on each `internal` equivalent. Usually for things that just "wrap" internal you don't need to, but since this is an alias, people can arrive at the internal version and not here and deserve to get the same docs.

Reply:
I see, I was wondering why there was so much cut/paste of docs in the codebase, but this explains...